proto/cosmos/group/v1beta1/query.proto

```diff
@@ -12,41 +12,41 @@ option go_package = "github.com/cosmos/cosmos-sdk/x/group";
 service Query {
 
   // GroupInfo queries group info based on group id.
-  rpc GroupInfo(QueryGroupInfoRequest) returns (QueryGroupInfoResponse);
+  rpc GroupInfo(QueryGroupInfo) returns (QueryGroupInfoResponse);
```

Comment:
I believe the convention is to remove `...Request` for Msgs, but keep for Queries